loveMyPet/loveMyPet/Modules/EditPet/ViewModel/EditPetViewModel.swift

```diff
@@ -76,30 +68,29 @@ final class EditPetViewModel: ObservableObject {
         }
     }
     
-    var formattedWeight: String {
-        return weight
-    }
+    //PESO
     
     func updateFormattedWeight() {
-        let kg = (weightKG != 0) ? "\(weightKG)" : ""
-        let g = (weightG != 0) ? "\(weightKG != 0 ? "," : "0,")\(weightG) kg" : "kg"
+        let kg = weightKG != 0 ? "\(weightKG)" : ""
```

Comment:
nessa linha `74` depois dos dois pontos não deveria ser `"0"`, já que no picker acredito que quando o peso não for selecionado mostrar algo e não apenas o `kg`

Reply:
Resolvido.